ted/ted/test.py

```diff
@@ -72,9 +72,16 @@ def __init__(self, fn, base, cfg):
 
         subdir = 'results'
 
+        samples_dir = config.get_samples_folder()
+
+        if samples_dir is None:
+            print(f"Samples were not found.")
```

Comment:
should not be here, it is a part of functions get_samples_folder()

it seems the second check for sample dir...